bin/picca_deltas.py

```diff
@@ -915,7 +924,7 @@ def main(cmdargs):
                                               delta.fiberid)
                     },
                     {
-                        'name': 'THING_ID',
+                       'name': 'THING_ID',
```

Comment:
shouldn't this raise an indentation error? I think this space should not be removed

Reply:
I'd run pylint to check what is the correct spacing